site/documentation/content/api/Tenant-API.md

```diff
@@ -152,36 +152,37 @@ The JSON structure below contains example information for tenant `TEST_TENANT`.
 
 ### Trusted CA Format
 
-The *trusted-ca* property of the tenant (JSON) payload can contain either a single JSON object containing the trusted CA to use for validating certificates of devices belonging to the tenant.
-+The tenant can also be configured with multiple trusted certificates. In this case, the *trusted-ca* property can contain an array of JSON objects, were each JSON object represents a single trusted CA. The table below provides an overview of the members defined in a JSON object contained in the *trust-ca* property:
+If the *trusted-ca* property is provided, then it MUST contain a non-empty array of JSON objects, were each JSON object represents a single trusted CA. The table below provides an overview of the members defined in a JSON object contained in the *trust-ca* property:
 
 | Name                     | Mandatory  | Type          | Default Value | Description |
 | :------------------------| :--------: | :------------ | :------------ | :---------- |
-| *subject-dn*             | *yes*      | *string*      |               | The subject DN of the trusted root certificate in the format defined by [RFC 2253](https://www.ietf.org/rfc/rfc2253.txt). |
-| *cert*                   | *no*       | *string*      |               | The Base64 encoded binary DER encoding of the trusted root X.509 certificate. |
+| *subject-dn*             | *yes*      | *string*      |               | The subject DN of the trusted root certificate in the format defined by [RFC 2253](https://www.ietf.org/rfc/rfc2253.txt). The subject DN MUST be unique across tenants. |
 | *public-key*             | *no*       | *string*      |               | The Base64 encoded binary DER encoding of the trusted root certificate's public key. |
-| *not-before*             | *no*       | *string*      |               | The property indicating that the trusted root X.509 certificate is not valid before the given date. |
-| *not-after*              | *no*       | *string*      |               | The property indicating that the trusted root X.509 certificate is not valid after the given date. |
+| *not-before*             | *no*       | *string*      |               | The property indicating that the trusted root X.509 certificate is not valid before the given date. The date format is defined by [RFC 5280](https://www.ietf.org/rfc/rfc5280.txt). |
```

Comment:
The X.509 RFC is not helpful here. See the definition of the data volume limit for the correct link/reference ...

Reply:
fix in 4380d8c